README.md

```diff
@@ -55,179 +30,140 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+- [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Capstone Project Module 1<a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**WTT Smash SINGAPORE 2023** is based on an online website for a conference according to design guidelines provided by Microverse. Trought this capstone project I show the main abilities adquired so far in the **Module 1 - The Portal**. In this Module I learned about HTML, CSS and an brief introduction to javaScript.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
-  <summary>Client</summary>
+  <summary>HTML</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li>HTML</li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>CSS</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li>CSS</li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>JavaScript</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li>JavaScritp</li>
   </ul>
 </details>
-
+<br>
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Creating branches on repositories.
+- Use of linter on HTML&CSS&js projects.
+- Write a correct descriptive README file.
+- Correct use of GitHub flow.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- Here you will see a [Live Demo](https://gdvl777.github.io/) of my project.
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your `Live Demo` link is not for this project [this link is for your `Portfolio`](https://gdvl777.github.io/), please make use of `GitHub Pages` to deploy the web pages 🙏🏼  The project is deployed properly, and all pages, links, and images work as expected.